Projects/Features/Scene/GroupDetailScene/MemberList/MemberListView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  MemberListView.swift
+//  GroupDetail
+//
+//  Created by 최혜린 on 6/22/24.
+//  Copyright © 2024 com.mashup.gabbangzip. All rights reserved.
+//
+
+import ComposableArchitecture
+import SwiftUI
+
+import DesignSystem
+import Models
+
+public struct MemberListView: View {
+  let store: StoreOf<MemberListCore>
+
+  public init(store: StoreOf<MemberListCore>) {
+    self.store = store
+  }
+
+  public var body: some View {
+    VStack(spacing: 0) {
+      NavigationBar(
+        type: .titleWithBackButton("그룹원"),
+        backButtonAction: {
+          store.send(.backButtonDidTap)
+        }
+      )
+      .padding(.bottom, 8)
+      
+      ForEach(store.memberList) { member in
+        MemberView(
+          member: member,
+          groupCategory: store.groupCategory
+        )
+      }
```

Comment:
```suggestion
      ForEach(store.memberList, id: \.self) { member in
        MemberView(
          member: member,
          groupCategory: store.groupCategory
        )
      }
```
`Member`가 equatable 대신 hashable을 채택한다면 위와 같이 작성할 수 있습니다. 그럼 멤버의 `id` 속성도 제거가 가능하답니다

Reply:
@zekexros 오 너무 좋은데요~~? 공부 더 해봐야겠습니다...감사합니다 🙇‍♀️